src/babel/build-external-helpers.js

```diff
@@ -17,7 +15,59 @@ export default function (whitelist) {
     )
   ]));
 
-  buildHelpers(body, namespace, whitelist);
+  builder(body);
+
+  return tree;
+}
+
+function buildUmd(namespace, builder) {
+  var body = [];
+  body.push(t.variableDeclaration("var", [
+    t.variableDeclarator(namespace, t.identifier("global"))
+  ]));
+
+  builder(body);
+
+  var container = util.template("umd-commonjs-strict", {
+    AMD_ARGUMENTS:      t.arrayExpression([t.literal("exports")]),
+    COMMON_ARGUMENTS:   t.identifier("exports"),
+    BROWSER_ARGUMENTS:  t.assignmentExpression("=", t.memberExpression(t.identifier("root"), namespace), t.objectExpression({})),
+    UMD_ROOT:           t.identifier("this"),
+    FACTORY_PARAMETERS: t.identifier("global"),
+    FACTORY_BODY:       body
+  });
+  return t.program([container]);
+}
+
+function buildVar(namespace, builder) {
+  var body = [];
+  body.push(t.variableDeclaration("var", [
+    t.variableDeclarator(namespace, t.objectExpression({}))
+  ]));
+  builder(body);
+  return t.program(body);
+}
+
+export default function (whitelist, outputType = "global") {
+  var namespace = t.identifier("babelHelpers");
+  var builder = function (body) {
+    return buildHelpers(body, namespace, whitelist);
+  };
+
+  var tree;
+  switch (outputType) {
+  case "global":
+    tree = buildGlobal(namespace, builder);
+    break;
+  case "umd":
+    tree = buildUmd(namespace, builder);
+    break;
+  case "var":
+    tree = buildVar(namespace, builder);
+    break;
+  default:
+    throw new Error("Unsupported output type");
```

Comment:
Add the message to [messages.js](https://github.com/babel/babel/blob/master/src/babel/messages.js) and then import `messages.js` and use `messages.get("unsupportedOutputType", outputType);`.